ros/src/sensing/drivers/camera/packages/vectacam/nodes/vectacam/VectaCam.h

```diff
@@ -0,0 +1,72 @@
+#ifndef VECTACAM_H_
+#define VECTACAM_H_
+
+#include <unistd.h>     /* defines STDIN_FILENO, system calls,etc */
+#include <sys/types.h>  /* system data type definitions */
+#include <sys/socket.h> /* socket specific definitions */
+#include <netinet/in.h> /* INET constants and stuff */
+#include <arpa/inet.h>  /* IP address conversion stuff */
+#include <netdb.h>      /* gethostbyname */
+
+#include <iostream>
+#include <fstream>
+#include <boost/algorithm/string.hpp>
+
+#include "opencv2/core/core.hpp"
+
+#define DEFAULT_BASE_PORT 	16386
+#define NUM_CAMERAS			8
+
+#define FRAME_START 		(0x01000000)
+#define FRAME_END   		(0x02000000)
+
+#define DATA_OFFSET 		(16)
+
+#define MAX_BUFFER 			1024*1024
+
+#define IMG_WIDTH			640
+#define IMG_HEIGHT			480
+
+#define CAMERA_IP			"10.0.0.1"
```

Comment:
These macros should start with `VC`, `VECTACAM` etc for avoiding confusion of other package macros. Or they should be declared as static class constant variable.